src/QCefView.cpp

```diff
@@ -330,15 +330,15 @@ QCefView::onRequestCloseFromWeb()
 void
 QCefView::leaveEvent(QEvent* event)
 {
-  QWidget::leaveEvent(event);
+  Q_D(QCefView);
 
-  if (d_ptr->isOSRModeEnabled()) {
-    QPoint localPos{ -1, -1 };
-    QPoint globalPos = mapToGlobal(localPos);
+  QPoint localPos = QCursor::pos();
+  QPoint globalPos = mapToGlobal(localPos);
```

Comment:
I think QCursor::pos() returns the position in global coordinate systems already.

https://doc.qt.io/qt-6/qcursor.html#pos